packages/docregistry/src/registry.ts

```diff
@@ -819,10 +819,21 @@ namespace DocumentRegistry {
   }
 
   /**
+   * A widget for a document.
+   */
+  export
+  interface IWidget extends Widget {
```

Comment:
This will eventually be in `rendermime-interfaces`, so let's define it here and use it in `docregistry` as `IReadyWidget`.